Makefile

```diff
@@ -64,10 +64,9 @@ pack:
 	echo "Validate package" && \
 	slim validate TA_dataset && \
 	echo "Preserve configuration" && \
-	if [ -f ./output/TA_dataset/local/ ]; then \
-  		mv -v ./output/TA_dataset/local/* $(CONFIGURATION_BACKUP); \
+	if [ -d ./output/TA_dataset/local/ ]; then \
+  		cp -v ./output/TA_dataset/local/* $(CONFIGURATION_BACKUP); \
   	fi && \
-	rm -rf output/TA_dataset && \
 	echo "Generate package" && \
 	ucc-gen --source TA_dataset && \
```

Comment:
Also change 
`ucc-gen --source TA_dataset`
to suggested
`ucc-gen build --source TA_dataset`

Reply:
Fixed